woocommerce/woocommerce-android
# Product subscriptions: Free trial section

Implements #10148, which adds a new Subscription free trial to the product detail and variation detail screen.

| Product detail | Subscription free trial |
| --- | --- |
| ![image](https://github.com/woocommerce/woocommerce-android/assets/1522856/38fcf858-4981-4390-a984-51a7090ea20e) | ![image](https://github.com/woocommerce/woocommerce-android/assets/1522856/055b5ea1-1790-49c2-9ec3-da9b24b0d91c) |

**To test:**

Simple subscription product
1. Make sure you have the Subscriptions plugin installed
2. Prepare a simple subscription product
3. Start the app
4. Go to Products -> Tap on the simple subscription product
5. Notice there's a new Subscription free trial section with the correct data shown
6. Tap on the free trial property
7. Notice a detail free trial screen is shown with the correct data
8. Try changing the values
9. Tap back
10. Notice the free trial section information was update

Variable subscription product
1. Make sure you have the Subscriptions plugin installed
2. Prepare a variable subscription product (make sure you generate some variations)
3. Start the app
4. Go to Products -> Tap on the variable subscription product
5. Tap on Variations -> Select a variation
6. Notice there's a new Subscription free trial section with the correct data shown
7. Tap on the free trial property
8. Notice a detail free trial screen is shown with the correct data
9. Try changing the values
10. Tap back
11. Notice the free trial section information was update
